app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -527,7 +527,6 @@
 
             & > .admin__field-label {
                 #mix-grid .column(@field-label-grid__column, @field-grid__columns);
-                background: @color-white;
```

Comment:
Why do we need to change the background color of all admin form fields to fix only one specific issue?

Reply:
Background color is not require there and "background: @color-white;" is deleted in 2.3.x. [link](https://github.com/magento/magento2/blob/1fb83b185b66f161dfa55744c23e213887f53fcd/app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less#L547)